services/mailer/mail.go

```diff
@@ -158,7 +158,8 @@ func SendRegisterNotifyMail(u *user_model.User) {
 		return
 	}
 
-	msg := NewMessage(u.Email, locale.TrString("mail.register_notify"), content.String())
+	appname := setting.AppName
+	msg := NewMessage(u.Email, locale.TrString("mail.register_notify", appname), content.String())
```

Comment:
The default value of `AppName` is not "Gitea" but "Gitea: Git with a cup of tea", I am not sure whether it is the right value for email title.

Reply:
I also thought about it, but it is most likely customized anyway, so probably not that big of a concern.